cli/install/install.go

```diff
@@ -1013,17 +1065,64 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 	}
 
 	// Check if program is already installed.
+	// If it is installed, check if newest version exists.
 	if !installCtx.Reinstall {
 		log.Infof("Checking existing...")
 		if util.IsRegularFile(filepath.Join(binDir, versionStr)) &&
 			util.IsDir(filepath.Join(incDir, versionStr)) {
-			log.Infof("%s version of tarantool already exists, updating symlinks...", versionStr)
-			err := changeActiveTarantoolVersion(versionStr, binDir, incDir)
+			isBinExecutable, err := util.IsExecOwner(filepath.Join(binDir, versionStr))
 			if err != nil {
 				return err
 			}
-			log.Infof("Done")
-			return err
+
+			var curBinVersion, lastCommitHash string
+
+			var answer bool
+			if isBinExecutable && tarVersion == "master" {
+				var err error
+				answer, err = util.AskConfirm(os.Stdin, "You already have installed "+
+					"tarantool_master.\nWant to check for newest versions?")
+				if err != nil {
+					return err
+				}
+			}
+
+			if answer {
```

Comment:
I've already seen similar code here. Can we extract this into a separate function?

Reply:
Extracted logic with checking latest version in separate function.